model-optimizer/mo/middle/passes/fusing/decomposition.py

```diff
@@ -39,8 +39,8 @@ def convert_batch_norm(graph: Graph):
     """
     nodes = graph.get_op_nodes()
     for node in nodes:
-        if node.has_valid('op') and (node.op in ['FusedBatchNorm', 'FusedBatchNormV2', 'FusedBatchNormV3',
-                                                 'BatchNorm', 'BatchNormalization', 'batchNormInference']):
+        if node.has_valid('op') and (node.op in ['BatchNorm', 'BatchNormalization', 
+                                        'batchNormInference']):
```

Comment:
nit: Merge into one line. It should fit into 120 characters limit.

Reply:
fixed